flexget/components/imdb/utils.py

```diff
@@ -15,7 +15,9 @@
 # IMDb delivers a version of the page which is unparsable to unknown (and some known) user agents, such as requests'
 # Spoof the old urllib user agent to keep results consistent
```

Comment:
Can we update the comment here to reflect the new situation?

Reply:
Oops, I meant the comment above, which is now wrong. I'll take care of it though.